### PR TITLE
Add list of Key Vault users

### PR DESCRIPTION
**Pull request by Stretch96**
* Allows providing a list of users that require access to the Key Vault tfvars secrets
* The values should be the User Principle Name found in AD

### CODE REVIEW

File: terraform/README.md [hunk 1]
```diff
@@ -159,6 +161,7 @@ If everything looks good, answer `yes` and wait for the new infrastructure to be
 | <a name="input_enable_mssql_database"></a> [enable\_mssql\_database](#input\_enable\_mssql\_database) | Set to true to create an Azure SQL server/database, with aprivate endpoint within the virtual network | `bool` | n/a | yes |
 | <a name="input_environment"></a> [environment](#input\_environment) | Environment name. Will be used along with `project_name` as a prefix for all resources. | `string` | n/a | yes |
 | <a name="input_image_name"></a> [image\_name](#input\_image\_name) | Image name | `string` | n/a | yes |
+| <a name="input_key_vault_access_users"></a> [key\_vault\_access\_users](#input\_key\_vault\_access\_users) | List if users that require access to the Key Vault where tfvars are stored. This should be a list of all users that need to run terraform | `list(string)` | n/a | yes |
```
Comment (mec) [inline terraform/README.md:164]: Type "List of"

File: terraform/terraform.tfvars.example [hunk 2]
```diff
@@ -10,3 +10,6 @@ container_command           = ["/bin/bash", "-c", "echo hello && sleep 86400"]
 container_environment_variables = {
   "ASPNETCORE_ENVIRONMENT" = "production"
 }
+key_vault_access_users = [
+  "someone_example.com#EXT#@tenantname.onmicrosoft.com",
```
Comment (mec) [inline terraform/terraform.tfvars.example:14]: I would like to understand where we get these 'Principle Names' in the documentation.

File: terraform/variables.tf [hunk 3]
```diff
@@ -3,6 +3,11 @@ variable "environment" {
   type        = string
 }
 
+variable "key_vault_access_users" {
+  description = "List if users that require access to the Key Vault where tfvars are stored. This should be a list of all users that need to run terraform"
```
Comment (mec) [inline terraform/variables.tf:7]: Same typo here.

**Review by mec — CHANGES_REQUESTED** [review]
Terrafrom looks good! Couple of housekeeping things.

File: terraform/variables.tf (hunk 3 above)
Comment (Stretch96) [reply terraform/variables.tf:7]: 👍

File: terraform/README.md (hunk 1 above)
Comment (Stretch96) [reply terraform/README.md:164]: 👍

**Review by mec — APPROVED** [review]
🚢

**Review by mec — APPROVED** [review]
🚢

**Review by mec — APPROVED** [review]
🚢

**Comment by mec** [comment]
🚢 🚢 🚢  I can't approve this but ship it!

**Pull request merged**